public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -64,35 +243,86 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
     );
   };
 
+  const formAPI = useForm<FormValues>({
+    mode: 'onSubmit',
+    defaultValues,
+    shouldFocusError: true,
+  });
+  const {
+    handleSubmit,
+    register,
+    watch,
+    formState: { isDirty, errors },
+  } = formAPI;
+
+  const onInvalid = () => {
+    notifyApp.error('There are errors in the form. Please correct them and try again!');
```

Comment:
Is this a message on the UI? If so-no please. Just Correct the errors and retry.

Reply:
yes, it's a message in the UI